presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -868,8 +868,7 @@ public synchronized void revokeRoles(Set<String> roles, Set<HivePrincipal> grant
 
     private String getTableOwner(HiveIdentity identity, String databaseName, String tableName)
     {
-        Table table = delegate.getTable(identity, databaseName, tableName)
```

Comment:
Can you do this refactoring change in a separate commit? It's hard to tell which ones are cleanup and which ones are new calls.